pySDC/core/Problem.py

```diff
@@ -135,3 +135,32 @@ def generate_scipy_reference_solution(self, eval_rhs, t, u_init=None, t_init=Non
 
         u_shape = u_init.shape
         return solve_ivp(eval_rhs, (t_init, t), u_init.flatten(), **kwargs).y[:, -1].reshape(u_shape)
+
```

Comment:
Why do these functions have to be part of the core problems class?

Reply:
I think it's a nice feature wrt prototyping to just do `prob.plot(u); plt.show()` and get a nice plot of the solution, complete with labels etc. In this case, for instance, you get plots of the two components side by side. I would like this to be implemented in other problems in the future as well. But I can remove it if you don't want it! 
Another advantage is that you can have hooks such as the one included in this PR, which you can just give to the controller and it plots the solution, as intended by the author of the problem class, in regular intervals.